src/components/scatterplot/ScatterplotSubscriber.js

```diff
@@ -85,6 +104,8 @@ export default function ScatterplotSubscriber(props) {
     SCATTERPLOT_DATA_TYPES,
   );
 
+  const title = titleOverride || `Scatterplot (${mapping})`;
```

Comment:
This is a very minor thing but now I am wondering if people will want to change only the "Scatterplot" part of the title and then they still want the "({mapping})" part to be appended?

I think for now we should keep it simple like this and I guess if we find it is a limitation we can always make a new feature request issue.

Reply:
Oh I think this might be erroneous - I must have missed deleting this when going through it. I think we should remove it and just go with what we have now.

Reply:
No it is not erroneous, I made this change because we want to be able to override component titles https://github.com/vitessce/vitessce/issues/843 (although it is a bit out of place in this specific PR but that is my fault)

Reply:
Ah ok my bad!